cmd/atlas/atlas.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/AlecAivazis/survey/v2/core"
-	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/root/atlas"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/root"
```

Comment:
won't this affect `go:generate mockgen ` references as well? 

both destination=. and package reference, ok if we want to do a follow-up PR but mock generation may be broken meanwhile so may be worth to get next PR ready

Reply:
>both destination=. and package reference, ok if we want to do a follow-up PR but mock generation may be broken meanwhile so may be worth to get next PR ready

No, I fixed the mocks and ran `make gen-mock` after my commit to confirm there are no changes as expected